components/brave_new_tab_ui/containers/newTab/index.tsx

```diff
@@ -1161,28 +1178,57 @@ class NewTabPage extends React.Component<Props, State> {
             </Page.GridItemBrandedLogo>}
             <FooterInfo
               textDirection={newTabData.textDirection}
-              onClickSettings={this.toggleSettings}
+              onClickSettings={this.openSettings}
               backgroundImageInfo={newTabData.backgroundImage}
               showPhotoInfo={!isShowingBrandedWallpaper && newTabData.showBackgroundImage}
             />
             </Page.FooterContent>
           </Page.Footer>
+          {newTabData.showToday &&
+          <Page.GridItemNavigationBraveToday>
+            <p>Scroll for Brave Today</p>
+            <div>
+              <CaratStrongDownIcon />
+            </div>
+          </Page.GridItemNavigationBraveToday>
+          }
         </Page.Page>
+        { newTabData.showToday &&
+        <BraveToday
+          feed={this.props.todayData.feed}
+          articleToScrollTo={this.props.todayData.articleScrollTo}
+          displayedPageCount={this.props.todayData.currentPageIndex}
+          publishers={this.props.todayData.publishers}
+          isFetching={this.props.todayData.isFetching === true}
+          isUpdateAvailable={this.props.todayData.isUpdateAvailable}
+          onRefresh={this.props.actions.today.refresh}
+          onAnotherPageNeeded={this.props.actions.today.anotherPageNeeded}
+          onInteracting={this.onBraveTodayInteracting}
+          // tslint:disable-next-line:jsx-no-lambda
+          onCustomizeBraveToday={() => { this.openSettings(SettingsTabType.BraveToday) }}
```

Comment:
nit: why can't this be `onCustomizeBraveToday={this.openSettings.bind(this, SettingsTabType.BraveToday)}`? Then no ts-lint-ignore needed

Reply:
More readable. Both are the same performance impact (i.e. none). I assumed that if we had `jsx-no-lambda` we also would have `jsx-no-bind`. I think we should just disable `jsx-no-lambda` globally since if we're down to the creation of an anonymous function having a performance impact we're either doing something very right or _very_ wrong!